### DISCUSSION TITLE
Failed to load mysqltcl - couldn't load library (PostgreSQL)均碰到了类似could not be found in library path问题

### DISCUSSION THREAD

**Discussion by zhangzhaox** [TPC-Council/HammerDB#455] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/82088551/200106776-29288f65-e36d-4e22-99d7-3322a6cfe898.png)
Error in Virtual User 1: Failed to load mysqltcl - couldn't load library "E:/HammerDB-4.5/lib/mysqltcl3.052/libmysqltcl.dll": this library or a dependent library could not be found in library path

**Comment by sm-shaw** [comment, marked as answer]
There might be some confusion here as the title says PostgreSQL but your problem is with MySQL.
This is fully explained in the documentation under the section [Verifying the Installation of Database Client Libraries ](https://www.hammerdb.com/docs/ch01s10.html) -  the example is exactly for your scenario. 
Install libmysql.dll and make sure it is in your path. The documentation should be your first recourse.
